text/0192-multi-search-api.md

```diff
@@ -0,0 +1,106 @@
+# Multi-search API
+
+## 1. Summary
+
+The multi-search endpoint performs a batch of search queries as if using the [search API](./0118-search-api.md) for an index.
+
+This endpoint allows, for example:
+
+- Execute different search queries on the same index
+- Execute search queries on different indexes
+
+Each search query has its own results set.
+
+## 2. Motivation
+
+- Perform multiple queries in a single HTTP request
+
+## 3. Functional Specification
+
+Meilisearch exposes 1 route to perform multi-search requests:
+
+- POST `/multi-search`
+
+If a master key is used to secure a Meilisearch instance, the auth layer returns the following errors:
+
+- 🔴 Accessing these routes without the `Authorization` header returns a [missing_authorization_header](0061-error-format-and-definitions.md#missing_authorization_header) error.
+- 🔴 Accessing these routes with a key that does not have permissions (i.e. other than the master key) returns an [invalid_api_key](0061-error-format-and-definitions.md#invalid_api_key) error.
+
+If any of the search queries fail to execute, the response returns the corresponding error instead of the array of results. If multiple queries would have failed, only the first encountered failure is returned.
```

Comment:
```suggestion
If any of the search queries fail to execute, the response returns the corresponding error instead of the array of results. If multiple queries fail, only the first encountered failure is returned.
```